src/index.tsx

```diff
@@ -8,10 +8,7 @@ import App from './app';
 import { configureStore } from './store';
 
 initApi({
-  host:
-    process.env.NODE_ENV === 'production'
-      ? 'http://koku-koku-dev.1b13.insights.openshiftapps.com'
-      : 'http://localhost:8000',
+  host: process.env.APP_NAMESPACE || 'http://localhost:8000',
```

Comment:
So `APP_NAMESPACE` is actually related to the Openshift project `koku-dev`, `koku-staging`, etc. So its a component of the url:
`'http://koku-' + process.env.APP_NAMESPACE + '.1b13.insights.openshiftapps.com'`

Let me know if that doesn't make sense.

Reply:
ah makes sense.  Updating. Will `https` vs `http` be an env setting in the near future?